Sources/Starknet/Data/StarknetChainId.swift

```diff
@@ -1,16 +1,20 @@
 import BigInt
 import Foundation
 
-public enum StarknetChainId {
-    case mainnet
-    case testnet
+public enum StarknetChainId: String, Codable, Equatable {
+    case mainnet = "0x534e5f4d41494e"
+    case goerli = "0x534e5f474f45524c49"
+    case sepolia_testnet = "0x534e5f5345504f4c4941"
+    case sepolia_integration = "0x534e5f494e544547524154494f4e5f5345504f4c4941"
```

Comment:
not sure we need this tbh.

Reply:
Probably won't hurt to have it just in case?

Reply:
does goerli integration introduce itself as goerli?

Reply:
Yes